CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 
 [Unreleased changes](./UNRELEASED.md).
 
+# 16.0.1
+
+**Fixed:**
```

Comment:
That's why rebase is better than merge 😛

Reply:
Lol. I'll update the base branch.